discovery-infra/update_assisted_service_cm.py

```diff
@@ -11,11 +11,12 @@
 #   MY_VAR: ""
 #
 # Hence, in order to support unset env vars, avoid override in Makefile.
-
+import json
 import os
-
 import yaml
 
+VALIDATOR_REQUIREMENTS = "HW_VALIDATOR_REQUIREMENTS"
```

Comment:
I think you can just inline this variable
This file is duplicating dictionary keys either way all over the place, and it's not that bad